src/main/java/it/cnr/isti/hpc/wikipedia/article/Link.java

```diff
@@ -92,7 +123,8 @@ public boolean isEmpty(){
 
 	@Override
 	public String toString() {
-		return "Link [id=" + id + ", description=" + getDescription() + "]";
+		return "Link [id=" + id + ", anchor=" + anchor + ", start=" + start + ", end=" + end + ", type=" + type
```

Comment:
To be honest I think this should method could only use the attributes `id`, `anchor`, `start`, `end` and `type`. I'm not 100% per cent sure, but it's mainly used for debugging purposes, so perhaps not worthy clunking it up with too much info.